qiskit/visualization/matplotlib.py

```diff
@@ -918,6 +925,15 @@ def _draw_ops(self, verbose=False):
                                       ac=self._style.dispcol['target'])
                     self._line(qreg_b, qreg_t, lc=lc)
 
+                # ccx gates
+                elif isinstance(op.op, ControlledGate) and op.name == 'ccx':
+                    num_ctrl_qubits = op.op.num_ctrl_qubits
+                    self._set_ctrl_bits(op.op.ctrl_state, num_ctrl_qubits,
+                                        q_xy, ec=ec, tc=tc, text=ctrl_text, qargs=op.qargs)
+                    self._x_tgt_qubit(q_xy[num_ctrl_qubits+1], ec=ec,
+                                      ac=self._style.dispcol['target'])
+                    self._line(qreg_b, qreg_t, lc=lc)
+
```

Comment:
I don't believe this elif is necessary. The ccx is covered by the `base_name == 'x'` elif above it.

Reply:
If I don't specifically add the elif here, the color would be in dark red instead of dark blue (please check the reference in #5032). The present version doesn't handle this scenario either, i.e. >=2 controlled qubit, the color is different from colors defined for swap and cx

Reply:
I see you added ccx to dispcol in qcstyle. This should take care of the color. I haven't run your code yet, but when I add it in to qcstyle with a different color using the master version, the color changes in the display output.